src/components/error/_macro.njk

```diff
@@ -2,19 +2,30 @@
 
 {% macro onsError(params) %}
     {% set content %}
-        <p
-            class="ons-panel__error"
+    <ul class="ons-list ons-list--bare">
+        {% if params.text is not string %}
+            {% for item in params.text %}
+            <li class="ons-panel__error ons-list__item"
+                {% if params.attributes %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %}{{ attribute }}{% if value %}="{{ value }}"{% endif %} {% endfor %}{% endif %}
+            >
+                <strong>{{ item }}</strong>
+            </li>
+            {% endfor %}
+        {% else %}
+        <li class="ons-panel__error ons-list__item"
             {% if params.attributes %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %}{{ attribute }}{% if value %}="{{ value }}"{% endif %} {% endfor %}{% endif %}
         >
             <strong>{{ params.text }}</strong>
-        </p>
+        </li>
```

Comment:
This should just stay as a `<p>` and be moved out of the `<ul>`. There isnt any real need for this to be a list if there is only one item and it could cause breaking changes in tests